packages/actor-extract-links-extract-tree/README.md

```diff
@@ -28,7 +28,12 @@ After installing, this package can be added to your engine's configuration as fo
     ...
     {
       "@id": "urn:comunica:default:extract-links/actors#extract-links-tree",
-      "@type": "ActorExtractLinksTree"
+      "@type": "ActorExtractLinksTree",
+      // If true, then during the traversal of TREE document
+      // regardless if the subject of the relation is the URL of the page
+      // we still consider the relations.
+      // This option exist because there are multiple TREE documents that doesn't respect this rule.
+      "loose": true
```

Comment:
Shall we name it `strict`? (and swap true/false)
That name is usually more common in these cases.

Reply:
I called it loose, because I wanted to put the emphasis that it is wrong to published data that way. If there were a strict mode for me it would send the message that, it's normal to publish data that way and it's out of "zeal" that we want to validate the URL.

Reply:
Yep, that's why I mentioned swapping true/false as well, so that `strict` mode is still the proper intended one.